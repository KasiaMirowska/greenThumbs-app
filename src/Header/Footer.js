import React from 'react';

export default function Footer () {
    return (
        <div className='footer'>
            <button>Back up</button>
        </div>
    )
}