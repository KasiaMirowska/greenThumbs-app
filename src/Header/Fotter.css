.footer {
    background-position: right bottom, left top; 
    background-size: cover; 
    /* background-size: 100%; */
    /* background-repeat: no-repeat; */
    /* background-image: url(https://green-images.s3.amazonaws.com/Image+URL.jpg) ;    */
    background-color: black;
    z-index: 1000;
    height: 5rem;
    display:flex;
    justify-content: flex-end;
}