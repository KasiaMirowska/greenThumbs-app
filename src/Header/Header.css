
.header {
    width: 100%;
    font-family: 'Anton', sans-serif;
    /* font-family: 'Oswald', sans-serif; */
    color: rgb(4, 195, 4);
    margin: 0;

    /* padding-left: 30px; */
}


.thumbs {
    color: orange;
     /* font-family: 'Indie Flower', cursive;  */
    /* font-family: 'Montserrat', sans-serif; */
}
.font {
    /* font-size: 3em;  */
    color: rgb(95, 177, 47);
    padding: 3px;
}
.h-text {
    line-height: 1.6;
    font-family: 'Montserrat', sans-serif;
    color: rgb(170, 195, 216);
}
span {
    color: orange;
}

@media (min-width: 600px) {
    .header {
        font-size: 5.5em;
        padding-top: 30px;
     
     } 
    main {
        width: 70%;
        margin: auto;
      }
    .h-text {
        width: 70%;
        margin-left: 30%;
    }
    }