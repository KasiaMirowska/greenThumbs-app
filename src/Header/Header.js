import React from 'react';
import { Link } from 'react-router-dom';
import TokenService from '../Services/token-service';
import config from '../config';
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faThumbsUp } from '@fortawesome/free-solid-svg-icons';
import './Header.css';


export default function Header(props) {

   
    return (
        <div>
            <h1 className='header'>
                <Link to='/'>
                    GREEN<span className='thumbs'>thumbs</span>UP
                {/* < FontAwesomeIcon icon={faThumbsUp} className='font' /> */}
                </Link>

                
            </h1>

            {(!TokenService.hasAuthToken(config.TOKEN_KEY)) ? <h3 className='h-text'>Welcome to GREEN thumbsUP! A place where you can create a positive impact on local restaurant industry. Search for a place to eat and let us know if that place adheres to enviroment friendly practices. Let's stand together for the Earth and influance business owners to take steps that will significatly lower their ecological footprint !
                <br />
                To post a review and browse through details of other user's reviews <a href={'/register'}>register </a> for the account or <a href={'/login'}>login to demo account </a>using <span>username : 'demo', password: 'demo12345'.</span>
            </h3> : null}
           
           
        </div>
    )
}