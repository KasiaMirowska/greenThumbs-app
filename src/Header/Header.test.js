import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import {BrowserRouter as BR} from 'react-router-dom';
import Header from './Header'

describe('List component', () => {
   
    it ('renders without crashing',() => {
        const div = document.createElement('div');
        ReactDom.render(<BR><Header /></BR>, div);
        ReactDom.unmountComponentAtNode(div);
    });
   
    it('renders UI as expected', () => {
        const item= renderer.create(<BR><Header /></BR>);
        expect(item.toJSON()).toMatchSnapshot();
    })
})