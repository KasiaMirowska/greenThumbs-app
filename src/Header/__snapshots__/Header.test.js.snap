// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component renders UI as expected 1`] = `
<div>
  <h1
    className="header"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      GREEN
      <span
        className="thumbs"
      >
        thumbs
      </span>
      UP
    </a>
  </h1>
  <h3
    className="h-text"
  >
    Welcome to GREEN thumbsUP! A place where you can create a positive impact on local restaurant industry. Search for a place to eat and let us know if that place adheres to enviroment friendly practices. Let's stand together for the Earth and influance business owners to take steps that will significatly lower their ecological footprint !
    <br />
    To post a review and browse through details of other user's reviews 
    <a
      href="/register"
    >
      register 
    </a>
     for the account or 
    <a
      href="/login"
    >
      login to demo account 
    </a>
    using 
    <span>
      username : 'demo', password: 'demo12345'.
    </span>
  </h3>
</div>
`;
